apps/opencs/model/world/columnimp.hpp

```diff
@@ -2417,7 +2434,125 @@ namespace CSMWorld
             return true;
         }
     };
-    
+
+    template<typename ESXRecordT>
+    struct TextureHandleColumn : public Column<ESXRecordT>
```

Comment:
Haven't finished studying the code so I am not quite sure yet what TextureHandleColumn is used for. But it sounds like that is the texture resource ID (basically the filename)? If yes, we should call it that, because I don't think we use the terminology "handle" anywhere else.

Reply:
From my understanding, it is only the name displayed to the user.

Reply:
Yeah, it seems so. Doesn't make any sense to me since LTEX records are identified by Index and not by ID. Looks redundant. Maybe we are missing something here.

@scrawl Do you have any further input regarding this topic.

Reply:
As far as I remember, yes, this field is redundant.